src/libs/antares/study/fwd.h

```diff
@@ -540,9 +540,28 @@ const char* DayAheadReserveManagementModeToCString(DayAheadReserveManagement daR
 */
 DayAheadReserveManagement StringToDayAheadReserveManagementMode(const AnyString& text);
 
+
+// ------------------------
+// MPS export status
+// ------------------------
+enum class mpsExportStatus : int
+{
+    NO_EXPORT = 0,
+    EXPORT_FIRST_OPIM = 1,
+    EXPORT_SECOND_OPIM = 2,
+    EXPORT_BOTH_OPTIMS = 3,
+    UNKNOWN_EXPORT = 4
+};
+
+std::string mpsExportStatusToString(const mpsExportStatus& mps_export_status);
+mpsExportStatus stringToMPSexportStatus(const AnyString& value);
+const char* mpsExportIcon(const mpsExportStatus& mps_export_status);
```

Comment:
Move this to **src/ui** ?

Reply:
Not sure we get a great benefit to separate declaration / definition of **mpsExportIcon** from the rest of the MPS stuff in fwd.h/.cpp.
We could place **mpsExportIcon** closer to the GUI, but it does not use wxWidget function, so ti's more a "utils" function. 
What we could do is (as it is done for unfeasible problem behavior) is regroup everything about MPS in mps_utils.h/.cpp (I tried this but I met problems so I gave up), or even make a little separate lib (**mps.lib**) to be linked wherever it's needed.

Reply:
I definitely that anything related to the GUI should be in **src/ui**, so for me it would be best to define this function as `static` in the source file where it is called.

Reply:
I moved **mpsExportIcon** into **src/ui**, where it is used. I didn't touch the 2 others functions for now to conform to the way it's done for other parameters.